lib/src/icons.dart

```diff
@@ -271,17 +296,27 @@ enum GYWIcons {
 
   /// Settings icon
   settings(
-    GYWIcon(
+    GYWIcon._library(
       name: "Settings",
       filename: "settings",
       width: 48,
       height: 48,
     ),
   ),
 
+  /// Unknown icon
+  unknown(
+    GYWIcon._library(
+      name: "Unknown",
+      filename: "unknown",
+      width: 48,
+      height: 48,
+    ),
+  ),
+
```

Comment:
This is not correct, the icon is not on the glasses. The image is only there for being used by the library when the icon is not found. We would have used `help` or something similar if we wished that.
```suggestion
```

Reply:
Or maybe we should just upload the "unknown" icon on the glasses as well for visual consistency?

Reply:
No, I 'd like to correctly separate the unknown icon from the other. Otherwise, a user has no insight to see if is an unknown image, or the actual unknown icon.

Reply:
How about now? I set the filename to "help".

Reply:
No, just remove the "unknown" icon from `GYWIcons`